service/dynamodb/internal/expclient/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/aws/aws-sdk-go-v2/service/dynamodb/internal/expclient
+
+go 1.14
+
+replace github.com/aws/aws-sdk-go-v2 => ../../../../
+
+replace github.com/aws/aws-sdk-go-v2/service/dynamodb => ../../
```

Comment:
Replace statements should not be committed

Reply:
hmm so we probably should have a discussion about this. I think ideally the entire SDK uses replace statements for all in repo code. And its probably not a stretch to make that same statement for the `smithy-go`. The only time replace statements will be used would be in development and testing.